File: ABC/061/A.py
import sys
A,B,C = map(int,input().split())

if A < -100 or A > 100 or B < -100 or B > 100 or C < -100 or C > 100:
    sys.exit()

if C >= A and C <= B:
    print("Yes")
else:
    print("No")

File: ABC/ruby/061/A.rb
A,B,C = gets().chomp.split(' ').map { |n| n.to_i }

exit if !( -100 <= A && A <= 100 && -100 <= B && B <= 100 && -100 <= C && C <= 100 )
puts A <= C && C <= B ? 'Yes' : 'No'